src/views/BuyMYC/BuyMYC.js

```diff
@@ -197,10 +197,10 @@ export default function BuyGMX() {
                   <div className="direct-purchase-options">
                     <Button
                       size="xl"
-                      imgSrc={gmxAvax}
+                      imgSrc={mycAvax}
                       href="https://traderjoexyz.com/trade?outputCurrency=0x62edc0692BD897D2295872a9FFCac5425011c661#/"
```

Comment:
This still links to buying the GMX token on Trader Joe

Reply:
Am going to leave since we dont have an MYC token on traderJoe. Think ill remove all network specific changes in a seperate pr